web/src/components/Incidents/IncidentsPage.jsx

```diff
@@ -169,6 +170,7 @@ const IncidentsPage = ({ customDataSource, namespace = '#ALL_NS#' }) => {
           setIncidentsData(processIncidents(aggregatedData));
           setFilteredData(filterIncident(filters, processIncidents(aggregatedData)));
           setIncidentsAreLoading(false);
+          dispatch(setIncidentsNavFilters('incidents', filters));
```

Comment:
you probably don't need the key as this can be set in the reducer
```suggestion
          dispatch(setIncidentsNavFilters(filters));
```